src/libraries/Common/src/System/Net/Http/aspnetcore/Http3/Frames/Http3ErrorCode.cs

```diff
@@ -75,7 +74,7 @@ internal enum Http3ErrorCode : long
         RequestCancelled = 0x10c,
         /// <summary>
         /// H3_REQUEST_INCOMPLETE (0x10E):
-        /// The client’s stream terminated without containing a fully-formed request.
+        /// The client?s stream terminated without containing a fully-formed request.
```

Comment:
Something odd happened to the file encoding here?

Reply:
Grr, I thought I caught all of those.  It should just be `'` rather than a smart tick.